### ISSUE TITLE
Continue button auto focuses to wrong element on iOS 17 voiceover

### ISSUE THREAD

**Issue by roshaanbajwa** [nhsuk/nhsuk-frontend#912]
# Bug Report

Continue button auto focuses to wrong element on iOS 17 voiceover. When the continue button is clicked, it doesn’t indicate anything to voiceover user that the focus state has been moved to the next page's continue button (as opposed to the page's header).

## What was the environment where this issue occurred?

* Device: iPhone
* Operating System: iOS 17

**Comment by roshaanbajwa** [comment]
https://github.com/nhsuk/nhsuk-frontend/assets/119668404/36b789cf-35e7-4c24-b44e-a91ac5ae90de

**Comment by andymantell** [comment]
This is a known issue in Voiceover unfortunately @roshaanbajwa - you will often find in Voiceover that it focuses on whatever is on the next page at around the same x,y location as the button that was clicked. It's been a while since I investigated this, but last time I did I came up with the following two links:

https://www.applevis.com/bugs/ios/voiceover-focus-may-spontaneously-jump-another-ui-element-most-typically-soon-after

and

https://lists.w3.org/Archives/Public/w3c-wai-ig/2015AprJun/0022.html

Basically this is a Voiceover bug, and I struggle to see how we would fix it without resorting to manually placing the focus onto something like the `<h1>` on page load, which feels like a very uncomfortable thing to be doing :-|

We dismissed it as "can't fix" on 111 online. But I'd be very interested to hear other's thoughts on this.

**Comment by roshaanbajwa** [comment]
Ah sad to hear it's just a voiceover bug - thanks for the update Andy
